cmake/CMakeLists.txt

```diff
@@ -779,28 +803,30 @@ if (onnxruntime_BUILD_BENCHMARKS)
   if (TARGET benchmark::benchmark)
     message("Use benchmark from preinstalled system lib")
   else()
-    message("Use benchmark from submodule")
     # We will not need to test benchmark lib itself.
     set(BENCHMARK_ENABLE_TESTING OFF CACHE BOOL "Disable benchmark testing as we don't need it.")
     # We will not need to install benchmark since we link it statically.
     set(BENCHMARK_ENABLE_INSTALL OFF CACHE BOOL "Disable benchmark install to avoid overwriting vendor install.")
-    add_subdirectory(${PROJECT_SOURCE_DIR}/external/googlebenchmark EXCLUDE_FROM_ALL)
```

Comment:
>   add_subdirectory(${PROJECT [](http://example.com/codeflow?start=1&length=29)

Not sure if you have seen this: [https://stackoverflow.com/questions/65527126/disable-install-for-fetchcontent](https://stackoverflow.com/questions/65527126/disable-install-for-fetchcontent)

Reply:
Also need to test how cmake install behaves here.

Reply:
I'm working on it.

Reply:
I made change that will keep the behavior.